### DISCUSSION TITLE
Welcome to flutter_chat Discussions!

### DISCUSSION THREAD

**Discussion by aliyazdi75** [aliyazdi75/flutter_chat#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 7flash** [comment]
I am building apps that makes experience of life even more satisfying, and flutter makes the whole development process really satisfying for me :smiley:

**Reply by aliyazdi75** [reply]
I'm so happy to hear that, Good Luck :smiley:
